pinot-server/src/main/java/org/apache/pinot/server/starter/helix/BaseServerStarter.java

```diff
@@ -680,6 +672,15 @@ public void start()
           Tracing.getThreadAccountant().getClusterConfigChangeListener());
     }
 
+    SendStatsPredicate sendStatsPredicate = SendStatsPredicate.create(_serverConf, _helixManager);
```

Comment:
We should not move this behind thread accountant initialization since server metrics need to be initialized.
cc @priyen-stripe who helped find the issue